### PR TITLE
Feature/performance module

### PR DESCRIPTION
**Pull request by Azmat7860**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -9,19 +9,21 @@ import hierarchyRoute from "./Hierarchy/index.js";
...
 const unProtectedRouter = express.Router();
 
 // Protected Routes
+protectedRouter.use("/performance", performanceRoute);
+protectedRouter.use("/misc", miscRoute);
+
 
 // Un-Protected Routes
 unProtectedRouter.use("/user", userRoute);
 unProtectedRouter.use("/employee", employeeRoute);
```
Comment (AbubakarZorrain) [inline routes/index.js:25]: It seems that your are not following today's msgs. @Azmat7860

Comment (AbubakarZorrain) [reply routes/index.js:25]: Make all routes private except signIn/signUp.

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

File: routes/index.js (hunk 1 above)
Comment (Azmat7860) [reply routes/index.js]: Sorry Sir 
But now it is done.
Please review it now

**Review by AbubakarZorrain — APPROVED** [review]
Approved...

**Pull request merged**